recipes/yieldfrom/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "yieldfrom" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yieldfrom-{{ version }}.tar.gz
+  sha256: eb75e58641410f06083f85013a438f28bceaabfe5af9c3206d5049362c37aca0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - yieldfrom
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Nurdok/yieldfrom/
+  summary: A backport of the `yield from` semantic from Python 3.x to Python 2.7
```

Comment:
This seems to contradict the `python >=3.6`? 

But I guess you still need this package as a dependency somewhere?

Reply:
Yes you're completely right. This is a way for us to get it working as we have right now. And refactor it using Conda environments.

Reply:
How about you make the requirement into `python >=2.7` then?

Reply:
Done :heavy_check_mark: